metadata/src/main/java/org/apache/kafka/controller/OffsetControlManager.java

```diff
@@ -36,7 +36,7 @@
 
 /**
  * Manages read and write offsets, and in-memory snapshots.
- *
+ * <br>
```

Comment:
nit: I think `<p>` (without `</p>`) is preferred over `<br>`

Reply:
TIL: https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#format
But the knowing enough HTML and semantic HTML this makes my brain itch. `<p>` is a tag that comes with the closing pair always and it's meant to embrace a paragraph. `<br>` is the one of the few tags that is allowed to be used without a closing one, and it's solely purpose is to add break lines.
But I see that Oracle recommends the use of `<p>`, so be it.